amundsen_application/static/js/features/ColumnList/index.tsx

```diff
@@ -325,6 +342,19 @@ const ColumnList: React.FC<ColumnListProps> = ({
     ];
   }
 
+  const openedColumnsMap = {};
+  const handleRowExpand = (rowValues) => {
+    if (openedColumnsMap[rowValues.name]) return;
+    openedColumnsMap[rowValues.name] = true;
+    logAction({
+      command: 'click',
+      label: `${rowValues.content.title} ${rowValues.type.type}`,
+      target_id: `column::${rowValues.content.title}`,
+      target_type: 'column stats',
+    });
```

Comment:
Our plan is to bit by by move out of this logging method and use a UI redux action to log things. 
See https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/ducks/ui/index.ts#L20

Reply:
Sorry, you would still keep the logAction for now!